modules/partition-replicator/src/integrationTest/java/org/apache/ignite/internal/partition/replicator/ItReplicaLifecycleTest.java

```diff
@@ -579,7 +579,7 @@ public void testTableEstimatedSize() throws Exception {
             });
 
             // Read the key from another transaction to trigger write intent resolution, and so incrementing the estimated size.
-            // TODO https://issues.apache.org/jira/browse/IGNITE-24360 Perhaps, it should be reworked some way
+            // TODO https://issues.apache.org/jira/browse/IGNITE-24361 Perhaps, it should be reworked some way
```

Comment:
Given code 
```
            node.transactions().runInTransaction(tx -> {
                keyValueView1.getAll(tx, kv1.keySet());

                keyValueView2.getAll(tx, kv2.keySet());
            });
```
is no longer needed. WI will be converted because of proper tx finish, explicit WI Resolution through gets not required.